Multisig/UI/App/MainTabBarViewController.swift

```diff
@@ -503,6 +503,11 @@ extension MainTabBarViewController: WebConnectionRequestObserver {
         guard !requestQueue.isEmpty && !presentingRequest else { return }
         let request = requestQueue.removeLast()
 
+        guard let connection = WebConnectionController.shared.connection(for: request), connection.status == .opened else {
```

Comment:
@DmitryBespalov Would be nice to add small comment here why is this needed

Reply:
ok